quipucords/scanner/satellite/six.py

```diff
@@ -481,6 +492,17 @@ def host_details(self, host_id, host_name):
         :returns: dictionary of host details
         """
         details = {}
+        sys_result = None
+        sys_results = self.inspect_result.systems.all()
```

Comment:
Same.  Seems like you can just filter here.

Reply:
done.